src/components/icon-list/icon-list.njk

```diff
@@ -0,0 +1,14 @@
+<ul class="usa-icon-list{% if iconColor %} usa-icon-list--{{ iconColor }}{% endif %}{% if iconSize %} usa-icon-list--size-{{ iconSize }}{% endif %}">
```

Comment:
Fixed

Reply:
In [Simple Content example](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/accelerator/3772-icon-list/components/detail/icon-list--simple.html) we have `usa-icon-list--primary`. But the icons in the list have `text-violet`.

![image](https://user-images.githubusercontent.com/3385219/110959396-4169c480-8313-11eb-9c6a-2bc9c8e1f6f4.png)

Is this intentional or should we do something like:
```njk
{% if not color and item.icon.color %}
  {% set icon_color = "text-" + item.icon.color %}
{% endif %}
<div class="usa-icon-list__icon {{ icon_color }}">
```

Reply:
Nope, not intentional. I'll fix

Reply:
I think I'll leave the component as-is, but remove the custom icon color from the YAML, because I think you _should_ be able to override a list-based variant with a more specific color. Now, for this to work properly you'd need your utilities to output with `!important`, which I don't need to do here. In this case, we don't need to show the overriding.